### DISCUSSION TITLE
Can I open a file from :Git?

### DISCUSSION THREAD

**Discussion by damiansan239** [tpope/vim-fugitive#1761] [Q&A]
I have NERDTree installed. So when open :Git it shows me a NERDTree like file tree. If I try to open a file from it gives me an error

![2021-05-29-062357_1366x768_scrot](https://user-images.githubusercontent.com/42095620/120059036-b57c5480-c046-11eb-8ec4-578fd890f8aa.png)

Please help, can it be possible?

**Comment by tpope** [comment]
I can confirm that NERDTree and Fugitive work fine together. Lots of people use both. My best guess is that some other plugin or your vimrc is getting in the way.
